### PR TITLE
ios pod update, nfc modal implementation, merge with stepper updates

### PR DESCRIPTION
**Pull request by merveillevaneck**

### CODE REVIEW

File: src/assets/images/ausweis_icon.png
Comment (BtencateSphereon) [inline src/assets/images/ausweis_icon.png:1]: i would suggest adding images as svg

File: src/localization/translations/en.json [hunk 1]
```diff
@@ -93,6 +93,9 @@
   "terms_of_service_decline_action_caption": "Exit app",
   "terms_of_service_title": "Terms of service",
   "terms_of_service_subtitle": "This document was last updated on January 5, 2023.",
+  "import_data_title": "Import personal data",
```
Comment (BtencateSphereon) [inline src/localization/translations/en.json:108]: where are the dutch localizations? it is fine adding them to the dutch file in english for now. so no need to translate it to dutch. but just make sure thy are present so we know which ones we still need to translate

File: src/screens/Onboarding/ImportPersonalDataSCreen/index.tsx [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+`;
+
+type AusweisEPinModalProps = {
+  initialValue?: string;
+  visible: boolean;
+  onComplete: (pin: string) => void;
+  focusOnMount?: boolean;
+};
+
+export const AusweisEPinModal = (props: AusweisEPinModalProps) => {
```
Comment (BtencateSphereon) [inline src/screens/Onboarding/ImportPersonalDataSCreen/index.tsx]: please move this to it's own file

File: src/screens/Onboarding/ImportPersonalDataSCreen/index.tsx [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+`;
+
+type AusweisScanModalProps = {
+  visible: boolean;
+  onComplete: () => void;
+  onCancel: () => void;
+  onScan: () => void;
+};
+
+export const AusweisScanModal = (props: AusweisScanModalProps) => {
```
Comment (BtencateSphereon) [inline src/screens/Onboarding/ImportPersonalDataSCreen/index.tsx]: please move this to it's own file

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]
the ticket is about implementing whole step 3, so we are missing some screens and implementation in this PR

**Comment by merveillevaneck** [comment]
closing because of syncing updates with newly merged `feat/new-onboarding` so rebased on top of funke and continued there addressing above comments as well as adding more UI

**Pull request merged**